upload.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# It reads latest uploaded version and compares it with currently built one.
+# If they are different, script will upload new jdk version and update file with latest name.
+set -eo pipefail
+
+echo "Checking $2..."
+latest=$(curl --silent -L http://packages.$2/buildpack-java/latest$1)
+echo "Latest version: $latest"
+
+current=$3
+echo "Current version: $current"
+
+if [ "$latest" == "$current" ]; then
```

Comment:
@fern4lvarez this can only happen if someone manually uploads some newer version which should be forbidden.
Moreover it is not that easy to distinguish which version is older than other because of the specific format of build numbering in openJDK project.